fixed saved search query (#27808)
github/docs: content/search-github/github-code-search/using-github-code-search.md

## Creating and managing saved searches

1. In the top navigation of {% data variables.product.prodname_dotcom_the_website %}, click the search bar and type `saved:`.
![GitHub top navigation, user typing 'saved:' in search bar.](/assets/images/help/search/navigatetosearch.png)

> 1. Under the search bar, the "Saved queries" section should now appear. Click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.

Reviewer: ```suggestion
1. Under the search bar, in the "Saved queries" section, click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
```

Small change to make this flow a little better and combine the two sentences.

!['Saved queries' section, user clicking 'Manage saved searches'.](/assets/images/help/search/selectmanagesavedsearch.png)

> 1. In the pop-up window, fill out the name you want for your query and the query that you want to save. Click **Create saved search**.

Reviewer: ```suggestion
1. In the pop-up window, type both the name you want for your saved search and the query you want to save.
1. To finish creating your saved search, click **Create saved search**.
```

Small change to some pre-existing text to better align with the [suggested terms](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#terms-to-use-or-avoid) in our style guide, and to split these actions into separate steps. Nothing to do with your edits! 😁

![Pop-up window, user creating saved search with name and query.](/assets/images/help/search/savedsearchquery.png)

> 1. If you click again on the search bar, you can now see your saved search in the "Saved queries" section under the search bar. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.

Reviewer: ```suggestion
1. To see your saved search, click the search bar. Your saved search will be in the "Saved queries" section. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
```

Just a couple more small changes to align this text with the [ordering recommended for procedural steps](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#procedural-steps).

![Search bar, 'Saved queries' section displaying saved search.](/assets/images/help/search/savedsearchviewinnavbar.png)

> 1. To edit or delete a saved search, type `saved:` in the search bar, and click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**. In the pop-up window to the right of the saved search, click {% octicon "pencil" aria-label="The pencil icon" %} to edit the search, or click {% octicon "trash" aria-label="The trash icon" %} to delete the search.

Reviewer: ```suggestion
1. To manage a saved search, type `saved:` in the search bar, then click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
    To edit a saved search, to the right of the search, click {% octicon "pencil" aria-label="The pencil icon" %}.
    To delete a saved search, to the right of the search, click {% octicon "trash" aria-label="The trash icon" %}.
```

A few more changes to break these steps up and follow the order for procedural steps described in the [style guide](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#procedural-steps) 🙂

## Creating and managing saved searches

1. In the top navigation of {% data variables.product.prodname_dotcom_the_website %}, click the search bar and type `saved:`.

> 
> 1. Under the search bar, in the "Saved queries" section, click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
> 
> 1. In the pop-up window, type both the name you want for your saved search and the query you want to save.
> 
> 1. To finish creating your saved search, click **Create saved search**.
> 
> 1. To see your saved search, click the search bar. Your saved search will be in the "Saved queries" section. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
> 
> 1. To manage a saved search, type `saved:` in the search bar, then click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
>    To edit a saved search, to the right of the search, click {% octicon "pencil" aria-label="The pencil icon" %}.
>    To delete a saved search, to the right of the search, click {% octicon "trash" aria-label="The trash icon" %}.

Reviewer: ```suggestion
1. Under the search bar, in the "Saved queries" section, click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
1. In the pop-up window, type both the name you want for your saved search and the query you want to save.
1. To finish creating your saved search, click **Create saved search**.
1. To see your saved search, click the search bar. Your saved search will be in the "Saved queries" section. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
1. To manage a saved search, type `saved:` in the search bar, then click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
    - To edit a saved search, to the right of the search, click {% octicon "pencil" aria-label="The pencil icon" %}.
    - To delete a saved search, to the right of the search, click {% octicon "trash" aria-label="The trash icon" %}.
```

Thanks for making those changes @npiyush97! I'll apply a couple final tweaks:

- we can remove those blank lines between the numbered steps.
- I added dashes to the substeps for managing saved searches to create an indented bulleted list. My apologies for missing that earlier!

I'll go ahead and commit these changes, then get this merged 🙂 thanks again for your help maintaining our docs! ✨
